site/pages/create-disruption/[disruptionId].page.tsx

```diff
@@ -244,17 +254,17 @@ const CreateDisruption = (props: DisruptionPageProps): ReactElement => {
     return (
         <BaseLayout title={title} description={description} errors={props.errors}>
             <CsrfForm
-                action={`/api/create-disruption${queryParams["template"] ? "?template=true" : ""}`}
+                action={`/api/create-disruption${getQueryParams(isTemplate === "true", returnPath)}`}
```

Comment:
While create a new disruption from a template after clicking save in create disruption page we should be navigate to select type of consequence but its redirecting to disruption details page